src/components/Gallery/Gallery.js

```diff
@@ -0,0 +1,33 @@
+import { ImageComponent } from '../ImageComponent/ImageComponent';
```

Comment:
Lack of the gallery label "Galeria" text from the designs, I know it's waiting for LabelComponent, you can add a TODO comment that after implementation of that we should add this Label here

Reply:
I used button component, tell me whether it is ok.

Reply:
It's not ok, button od clickable, label is label

Reply:
Done.